src/main/kotlin/com/colivery/serviceaping/persistence/entity/HelpSeekerEntity.kt

```diff
@@ -0,0 +1,31 @@
+package com.colivery.serviceaping.persistence.entity
+
+import com.colivery.serviceaping.persistence.Source
+import org.hibernate.annotations.Type
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import java.util.*
+import javax.persistence.*
+
+@Entity
+@Table(name = "`help_seeker`")
+@EntityListeners(AuditingEntityListener::class)
+data class HelpSeekerEntity(
+        @ManyToOne(optional = true, fetch = FetchType.EAGER)
+        var user: UserEntity,
+
+        var fullName: String? = null,
+
+        var phone: String? = null,
+
+        @Column(nullable = false)
+        @Type(type = "com.vladmihalcea.hibernate.type.basic.PostgreSQLEnumType")
+        @Enumerated(EnumType.STRING)
+        var source: Source,
```

Comment:
`Source` currently contains only `HOTLINE` and `APP`. I think none of those really fit the admin dashboard, right? Maybe we could add the third option as well? Could need additional migrations if the `Source` enum is used in database already.